src/app/screens/LeaveDetailScreen.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {
+  SafeAreaView,
+  StyleSheet,
+  View,
+  ActivityIndicator,
+  ScrollView,
+} from 'react-native';
+
+import Header from '../components/header';
+import Typography from '../components/typography';
+import {useLeaveDetail} from './LeaveScreen/leave.hooks';
+
+import colors from '../constant/colors';
+import {LeaveDetailScreenNavigationProp} from '../navigation/types';
+import DetailRow from './ProfileScreen/component/DetailRow';
+
+function LeaveDetailScreen({route}: LeaveDetailScreenNavigationProp) {
+  const {leaveID} = route.params;
+
+  const {data, isLoading} = useLeaveDetail(leaveID);
+
+  const {
+    emp_name,
+    leave_from,
+    leave_to,
+    leave_type,
+    leave_approver,
+    leave_note,
+    leave_reason,
+    leave_status,
```

Comment:
use alises and convert as per js standards
```suggestion
    emp_name: empName,
    leave_from: leaveFrom,
    leave_to,
    leave_type,
    leave_approver,
    leave_note,
    leave_reason,
    leave_status,
```